recipes/offset_monitor.rb

```diff
@@ -101,6 +101,27 @@
   notifies :restart, "service[kafka-offset-monitor]"
 end
 
+# Write JAAS configuration file if enabled
+if node["kafka"]["kerberos"]["enable"]
+
+  jaas_path = "#{node['kafka']['offset_monitor']['install_dir']}/jaas.conf"
+
+  # add JAAS config location as default JVM parameter
+  node.default['kafka']['offset_monitor']['java_options']['-Djava.security.auth.login.config='] = jaas_path
```

Comment:
thoughts on setting this in the attributes instead? also may want to consider delaying evaluation of its derived value